integrations/access/email/config.go

```diff
@@ -66,45 +66,6 @@ type Config struct {
 	Log              logger.Config           `toml:"log"`
 }
 
-const ExampleConfig = `# Example email plugin configuration TOML file
```

Comment:
This exported variable is used by `access/email` in teleport-plugins: https://github.com/gravitational/teleport-plugins/blob/master/access/email/main.go#L55